### PR TITLE
VA-94 updated inbox day and header graphic

### PR DESCRIPTION
**Pull request by shewood**
[VA-94](https://jira-dev.bdm-dev.dts-stn.com/browse/VA-94)
[VA-67](https://jira-dev.bdm-dev.dts-stn.com/browse/VA-67)

Updated the font styling of the day in the inbox.
Changed the canada logo to be english first for en and french first for fr.

### CODE REVIEW

File: src/components/molecules/TheHeader.vue [hunk 1]
```diff
@@ -117,6 +128,9 @@ export default {
       useI18n().locale.value == "fr" ? "en" : "fr"
     );
     return { changeLanguageTo };
+
+    const lang = computed(() => (useI18n().locale.value == "fr" ? "en" : "fr"));
+    return { lang };
```
Comment (TishamIslam) [inline src/components/molecules/TheHeader.vue:133]: Doesn't seem like you used this.

**Review by TishamIslam — CHANGES_REQUESTED** [review]
Works great! Just one small thing that you should change.

File: src/components/molecules/TheHeader.vue [hunk 2]
```diff
@@ -41,16 +41,27 @@
...
         <div
           className="flex flex-col sm:flex-row container w-full mx-auto py-1 px-5 justify-between items-center"
         >
           <div className="flex flex-row items-center justify-between">
-            <img
-              className="h-5 xs:h-7 sm:h-8 md:h-9 mx-3 xs:mx-0"
-              src="../../assets/sig-blk-en.svg"
-              :alt="$t('canadaLogoHeader')"
-            />
+            <div v-if="changeLanguageTo === 'fr'">
```
Comment (nibivi77) [inline src/components/molecules/TheHeader.vue:50]: This is exactly what I was thinking :) the only thing is that I would remove both `<div>` and place the v-if and v-else on the `img` tag themselves.

**Review by nibivi77 — CHANGES_REQUESTED** [review]
One small change :) This is great!

File: src/components/molecules/TheHeader.vue (hunk 2 above)
Comment (shewood) [reply src/components/molecules/TheHeader.vue:50]: I will try that, I wasn't sure.

File: src/components/molecules/TheHeader.vue (hunk 1 above)
Comment (shewood) [reply src/components/molecules/TheHeader.vue:133]: good catch. I will remove it as it isn't used.

**Review by nibivi77 — APPROVED** [review]
Looks good to me!

**Review by TishamIslam — APPROVED** [review]

**Pull request merged**